lib/ood_core/job/adapters/kubernetes/batch.rb

```diff
@@ -176,6 +176,10 @@ def group
     Etc.getgrgid(run_as_group).name
   end
 
+  def supplemental_groups
+    OodSupport::User.new.groups.sort_by(&:id).map(&:id)
```

Comment:
We should definitely `map(&:id).reject { |id| id < 1024 }`.  If a system has a minimum of 500 UID, does the same apply to group ids?

Reply:
So I did `id < 1000` because at least at OSC and the default for RHEL7+ is to do user UID/GID as 1000+.  The default for RHEL6 was 500+ but I believe during migration to OSCUS, we moved all UIDs/GIDs below 1000.

Really the minimum UID/GID is really not enforced much by anything.  There are places that configure like `useradd` and `groupadd` commands that when UID/GID is not specified, it picks from a range depending on if the record created is flagged as system account. For RHEL7+ if you tell `useradd` or `groupadd` to create a system record it will be less than 1000.  For OSC and others using LDAP, it really just comes down to standards of what is created in LDAP, since I could easily create a group with GID 200 and it is treated not differently than group with GID 2000.